src/Server/Remote/NativeSsh.php

```diff
@@ -141,23 +99,25 @@ public function scpCopy($target, $target2)
         $scpOptions = [];
 
         if ($serverConfig->getConfigFile()) {
-            $scpOptions[] = '-F ' . escapeshellarg($serverConfig->getConfigFile());
+            $scpOptions = array_merge_recursive($scpOptions, ['-F' => $serverConfig->getConfigFile()]);
         }
 
         if ($serverConfig->getPort()) {
-            $scpOptions[] = '-P ' . escapeshellarg($serverConfig->getPort());
+            $scpOptions = array_merge_recursive($scpOptions, ['-P' => $serverConfig->getPort()]);
         }
 
         if ($serverConfig->getPrivateKey()) {
-            $scpOptions[] = '-i ' . escapeshellarg($serverConfig->getPrivateKey());
+            $scpOptions = array_merge_recursive($scpOptions, ['-i' => $serverConfig->getPrivateKey()]);
+        } elseif ($serverConfig->getPemFile()) {
```

Comment:
I've noticed that pem file support for ssh connection was added recently but it was missed for scp.